### PR TITLE
Add custom stopping power

### PR DESCRIPTION
**Pull request by ostatni5**

### CODE REVIEW

File: src/util/CounterMap/CounterMap.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export class CounterMap<K extends string> {
 		return newValue;
 	}
 
+	size() {
+		return this.map.size;
+	}
```
Comment (Derstilon) [inline src/util/CounterMap/CounterMap.ts:46]: image
Remove 👁️ 👁️ ?

File: src/types/SimulationProperties.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ export type SimulationPropertiesType = {
 	readonly notRotatable?: boolean;
 	readonly notScalable?: boolean;
 	readonly notHidable?: boolean;
-	readonly notCloneable?: boolean;
+	readonly notDuplicatable?: boolean;
```
Comment (Derstilon) [inline src/types/SimulationProperties.ts:15]: Wierd change but okay

**Review by Derstilon — APPROVED** [review]

File: src/util/CounterMap/CounterMap.ts (hunk 1 above)
Comment (Derstilon) [inline src/util/CounterMap/CounterMap.ts:46]:
```suggestion
```

**Review by Derstilon — APPROVED** [review]

**Review by grzanka — APPROVED** [review]
Works well, needs https://github.com/yaptide/converter/issues/134 for full testing. I'd merge it as it is and then fix any possible bugs once the converter is ready.

**Pull request merged**